cmd/controller-manager/main.go

```diff
@@ -54,6 +55,73 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const defaultNamespace = "kube-system"
+
+// ensureCRDExists to ensure CRD exists before start
+func ensureCRDExists(kubeClient kubernetes.Interface, clientSet *versioned.Clientset) error {
+	condition := func() (bool, error) {
+		var err error
+		_, err = kubeClient.CoreV1().Namespaces().Get(defaultNamespace, metav1.GetOptions{})
+		if err == nil {
+			// Only namespace "kube-system" exists, to list resources
+			_, err = clientSet.PingcapV1alpha1().Backups(defaultNamespace).List(metav1.ListOptions{})
```

Comment:
Can we check CRD scheme instead of listing resources here?

Reply:
I was also thinking of checking the CRD directly but that will require new rbacs.

Reply:
OK. you may add some comments to explain this. 😂

Reply:
good suggestion.